plan/analyze_test.go

```diff
@@ -66,6 +74,14 @@ func (s *testAnalyzeSuite) TestAnalyze(c *C) {
 			sql:  "select * from t1 where t1.a = 1 and t1.b <= 2",
 			best: "Index(t1.a)[[1,1]]",
 		},
+		{
+			sql:  "select * from t2 where t2.a <= 2",
+			best: "Index(t2.b)[[<nil>,+inf]]",
```

Comment:
The result is obvious NOT the best.
Explain why, should we fix it now or add FIXME comment?